tests/test_web_sendfile_functional.py

```diff
@@ -751,3 +752,68 @@ def test_static_route_path_existence_check() -> None:
     assert 'application/octet-stream' == resp.headers['Content-Type']
     assert resp.headers.get('Content-Encoding') == 'deflate'
     await resp.release()
+
+
+async def test_static_file_huge_cancel(aiohttp_client, tmpdir) -> None:
+    filename = 'huge_data.unknown_mime_type'
+
+    # fill 100MB file
```

Comment:
just this:

```python
with tmpdir.join(filename).open('wb') as f:
    f.seek(100*1024*1024)
    f.write(b'1')
```

Reply:
this will efficiently create big (but sparse) file. Since that, You may safely increse size to, say, 1Gb. It will not lead to disk IO in any FS except FAT. But no one will run test on FAT fs.